### PR TITLE
[Chore] Update/Organize README 

### PR DESCRIPTION
**Pull request by kittinunf**
## Description

This PR updates outdated README files in the project.

- [x] Update dependency versions
- [x] Remove some of the duplicate entries so it reduces the maintenance costs
- [x] Update README of subprojects.

## Type of change

Check all that apply

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactoring (a change which changes the current internal or external interface)
- [x] This change requires a documentation update

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation, if necessary
- [x] My changes generate no new compiler warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Inspect the bytecode viewer, including reasoning why

### CODE REVIEW

**Comment by SleeplessByte** [comment]
LGTM!

**Comment by kittinunf** [comment]
@SleeplessByte approve please? 🙏

File: README.md [hunk 1]
```diff
@@ -61,16 +67,20 @@ repositories {
 }
 
 dependencies {
+  //core
   implementation(group = "com.github.kittinunf.fuel", name = "fuel", version = "-SNAPSHOT")
-  implementation(group = "com.github.kittinunf.fuel", name = "fuel-coroutines", version = "-SNAPSHOT")
-  implementation(group = "com.github.kittinunf.fuel", name = "fuel-kotlinx-serialization", version = "-SNAPSHOT")
+  
+  //packages
+  implementation(group = "com.github.kittinunf.fuel", name = "<package>", version = "-SNAPSHOT")
```
Comment (SleeplessByte) [inline README.md]:
```suggestion
  // replace <package> with the package name e.g. fuel-coroutines
  implementation(group = "com.github.kittinunf.fuel", name = "<package>", version = "-SNAPSHOT")
```

File: README.md [hunk 2]
```diff
@@ -61,16 +67,20 @@ repositories {
...
 }
 ```
 
+or
 
 ```kotlin
 dependencies {
-  listof("fuel", "fuel-coroutines", "fuel-kotlinx-serialization").forEach {
+  //core and/or packages
+  listof("fuel", "<package>").forEach {
```
Comment (SleeplessByte) [inline README.md]:
```suggestion
  // replace <package> with the package name e.g. fuel-coroutines
  listof("fuel", "<package>").forEach {
```

**Pull request merged**